azure-pipelines.yml

```diff
@@ -33,18 +33,30 @@ jobs:
       maxParallel: 6
       matrix:
         Linux.Debug:
-          image_name: 'ubuntu-18.04'
+          image_name: 'ubuntu-20.04'
```

Comment:
I agree we need the coverage - and let's take this in for now BUT I wonder if we could trim down slightly - maybe do debug on one platform and RelWithDebInfo on the other or something to reduce the CI run time.